.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,21 @@
-### What does this PR do?
+Here are three templates below, please choose one of them and delete others.
+Put correct title based on the title template(scope with parentheses is optional), and keep only the PR body template here.
+-----------------------------------------------------------------------------------------
+**if this is a feature work, add the technical writer as a reviewer to the PR**
```

Comment:
There is some formatting missing in the rendered version https://github.com/forcedotcom/salesforcedx-vscode/blob/c2b596cf59b67bd027a2c38cdba478cacb4bb9a5/.github/PULL_REQUEST_TEMPLATE.md
Can you please check and fix.

Reply:
@mohanraj-r  I also thought about the rendered version of this file in the first place, but decided to ignore it since we developer only will look at the original version. Besides, only the template body will be kept and that part is well formatted.